Classes/Core/SystemEnvironmentBuilder.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
```

Comment:
I propose we make new files strict to begin with.

Reply:
Oops